### PR TITLE
8334106: Problemlist CAInterop.java#quovadisrootca1g3 due to JDK-8334105

### PR DESCRIPTION
**Pull request by sendaoYan**
Hi all, 
  Test `security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java#quovadisrootca1g3` report failure as `failed to validate`, before the validate issue has been fixed, should we problem list the testcase.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8334106](https://bugs.openjdk.org/browse/JDK-8334106): Problemlist CAInterop.java#quovadisrootca1g3 due to JDK-8334105 (**Sub-task** - P3)

### Reviewers
 * [Christoph Langer](https://openjdk.org/census#clanger) (@RealCLanger - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19685/head:pull/19685` \
`$ git checkout pull/19685`

Update a local copy of the PR: \
`$ git checkout pull/19685` \
`$ git pull https://git.openjdk.org/jdk.git pull/19685/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19685`

View PR using the GUI difftool: \
`$ git pr show -t 19685`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19685.diff">https://git.openjdk.org/jdk/pull/19685.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19685#issuecomment-2164202057)

### CODE REVIEW

**Review by RealCLanger — APPROVED** [review]
We see the same at SAP. I welcome this exclusion.

**Comment by sendaoYan** [comment]
Thanks for the review and approved.
/integrate

**Comment by sendaoYan** [comment]
> let's wait a bit with integration before the security folks had a chance to look at this...

Okey.

**Comment by seanjmullan** [comment]
> let's wait a bit with integration before the security folks had a chance to look at this...

Thanks. Please wait for @rhalade to review.

**Comment by sendaoYan** [comment]
Thanks for the approved.

**Comment by seanjmullan** [comment]
@sendaoYan  As a best practice, it would be useful to first understand why the test is not working before putting it on the ProblemList. Depending on the severity of the problem that is not always possible, but it should be the first step in an evaluation in my opinion. Minimally, the referenced issue should have an Assignee so that it is assured someone will look into it. Once an issue is on the ProblemList, it unfortunately doesn't address the underlying issue, and may not get the same attention if some evaluation had been done beforehand. Another suggestion is to ask about it on the OpenJDK security-dev alias, where there are Security Group members who have more experience with these tests and can decide what the best course of action is.

The infra tests are not part of any of our CI tiers, so generally there is more time to investigate and figure out what the issue is in tests like this before putting it on the Problem List. Often the certificate tests fail because of an issue on the CA side, which sometimes can be fixed quickly after contacting the CA.

In summary, please hold off on adding this test to the ProblemList until we have some time to evaluate the test failure. Thank you.

**Comment by sendaoYan** [comment]
> @sendaoYan As a best practice, it would be useful to first understand why the test is not working before putting it on the ProblemList. Depending on the severity of the problem that is not always possible, but it should be the first step in an evaluation in my opinion. Minimally, the referenced issue should have an Assignee so that it is assured someone will look into it. Once an issue is on the ProblemList, it unfortunately doesn't address the underlying issue, and may not get the same attention if some evaluation had been done beforehand. Another suggestion is to ask about it on the OpenJDK security-dev alias, where there are Security Group members who have more experience with these tests and can decide what the best course of action is.
> 
> The infra tests are not part of any of our CI tiers, so generally there is more time to investigate and figure out what the issue is in tests like this before putting it on the Problem List. Often the certificate tests fail because of an issue on the CA side, which sometimes can be fixed quickly after contacting the CA.
> 
> In summary, please hold off on adding this test to the ProblemList until we have some time to evaluate the test failure. Thank you.

Got it. Thank you for your detailed explanation.
If this issue fails cause by CA side or some other reason, and the the failure can fixed quickly, I think we should close these related PRs.

**Comment by ArnoZeller** [comment]
> The infra tests are not part of any of our CI tiers, so generally there is more time to investigate and figure out what the issue is in tests like this before putting it on the Problem List. Often the certificate tests fail because of an issue on the CA side, which sometimes can be fixed quickly after contacting the CA.

@seanjmullan : I am a bit astonished that the CAInterop tests are not part of any of your CI tiers. We do run the JDK tier 4 JTreg tests (that contains everything expect tier 1 , 2 and 3) on a regular time schedule and these CAInterop tests cause a lot of noise in our testing environment. Do you think you could add the CAInterop test to your automated tests too - or on the other hand, perhaps they could be simply marked manually or intermittent to avoid automated execution?

**Comment by seanjmullan** [comment]
> > The infra tests are not part of any of our CI tiers, so generally there is more time to investigate and figure out what the issue is in tests like this before putting it on the Problem List. Often the certificate tests fail because of an issue on the CA side, which sometimes can be fixed quickly after contacting the CA.
> 
> @seanjmullan : I am a bit astonished that the CAInterop tests are not part of any of your CI tiers. We do run the JDK tier 4 JTreg tests (that contains everything expect tier 1 , 2 and 3) on a regular time schedule and these CAInterop tests cause a lot of noise in our testing environment. Do you think you could add the CAInterop test to your automated tests too - or on the other hand, perhaps they could be simply marked manually or intermittent to avoid automated execution?

Our internal infrastructure restricts external connections. We normally run these tests manually before integrating changes in the security libs area.

**Comment by RealCLanger** [comment]
@sendaoYan Looks like the root cause was fixed by the CA and this PR (and according bug) can be closed now.

**Comment by sendaoYan** [comment]
> @sendaoYan Looks like the root cause was fixed by the CA and this PR (and according bug) can be closed now.

Okey.
